client/src/pages/CreateSurvey/Questions.jsx

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  TextField,
+  Button,
+  Select,
+  InputLabel,
+  FormControl,
+  MenuItem,
+  Checkbox,
+} from '@material-ui/core';
+
+const Option = ({ optionIndex, questionIndex }) => {
+  const dispatch = useDispatch();
+  const { questions } = useSelector((state) => state.createSurveyReducer);
+
+  const { options } = questions[questionIndex];
+  const { text } = options[optionIndex];
+
+  const setOptionText = (event) => {
+    const payload = {};
+    payload.questionIndex = questionIndex;
+    payload.optionIndex = optionIndex;
+    payload.text = event.target.value;
+    dispatch({ type: 'SET_OPTION_DATA', payload });
+  };
+
+  const handleDeleteOption = () => {
+    const payload = {};
+    payload.questionIndex = questionIndex;
+    payload.optionIndex = optionIndex;
+    dispatch({ type: 'DELETE_OPTION', payload });
+  };
+  return (
+    <>
+      <TextField
+        required
+        error={text.length < 1}
+        helperText={text.length < 1 ? 'Answer must not be empty!' : ''}
```

Comment:
`text.length < 1 && 'Answer must not be empty!'` makes more sense!

Reply:
agreed

Reply:
This is great, love the comments! Don't forget to go through and resolve them as they are fixed so that it is clear what exactly needs to be worked on :)